doc/policies_list/3.3.0/policies.json

```diff
@@ -0,0 +1,1276 @@
+          "accuracy in the rate limits."
+        ],
+        "name": "3scale batcher",
+        "summary": "Caches auths from 3scale backend and batches reports.",
+        "version": "builtin"
+      }
+    ],
+    "3scale_referrer": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {},
+          "type": "object"
+        },
+        "description": "Sends the 'Referer' to 3scale backend so it can be validated",
+        "name": "3scale Referrer",
+        "summary": "Sends the 'Referer' to 3scale backend so it can be validated.",
+        "version": "builtin"
+      }
+    ],
+    "apicast": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {},
+          "type": "object"
+        },
+        "description": [
+          "Main functionality of APIcast to work with the 3scale API ",
+          "manager. This includes matching of mapping rules, authorization, ",
+          "reporting, etc."
+        ],
+        "name": "3scale APIcast",
+        "summary": "Main functionality of APIcast to work with the 3scale API manager.",
+        "version": "builtin"
+      }
+    ],
+    "caching": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {
+            "caching_type": {
+              "default": "none",
+              "description": "Caching mode",
+              "oneOf": [
+                {
+                  "enum": [
+                    "strict"
+                  ],
+                  "title": "Strict: cache only authorized calls."
+                },
+                {
+                  "enum": [
+                    "resilient"
+                  ],
+                  "title": "Resilient: authorize according to last request when backend is down."
+                },
+                {
+                  "enum": [
+                    "allow"
+                  ],
+                  "title": "Allow: when backend is down, allow everything unless seen before and denied."
+                },
+                {
+                  "enum": [
+                    "none"
+                  ],
+                  "title": "None: disable caching."
+                }
+              ],
+              "type": "string"
+            }
+          },
+          "type": "object"
+        },
+        "description": [
+          "Configures a cache for the authentication calls against the 3scale ",
+          "backend. This policy support four kinds of caching: \n",
+          " - Strict: it only caches authorized calls. Denied and failed calls ",
+          "invalidate the cache entry.\n",
+          " - Resilient: caches authorized and denied calls. Failed calls do not ",
+          "invalidate the cache. This allows us to authorize and deny calls ",
```

Comment:
@davidor who is "us" in _This allows us to authorize and deny calls_? For example, does it refer to 3scale AMP?